src/main/java/nl/tudelft/dnainator/ui/drawables/phylogeny/InternalNode.java

```diff
@@ -34,47 +34,24 @@ public InternalNode(List<AbstractNode> children) {
 			child.translateXProperty().set(LEVELWIDTH);
 			rangeBegin = rangeBegin.add(child.marginProperty());
 			Edge e = new Edge(child);
-			this.getChildren().add(0, e);
-			child.setIncomingEdge(e);
+			this.outgoingEdges.add(e);
 		}
+		this.getChildren().addAll(0, outgoingEdges);
 		// Add the nodes after the edges, so that they're on top.
 		this.getChildren().addAll(children);
-		bindInactiveProperties();
 	}
 
-	private void bindInactiveProperties() {
-		inactiveProperty().bind(
-				Bindings.createBooleanBinding(() -> children.stream()
-						.allMatch(AbstractNode::getInactive), children.stream()
-						.map(AbstractNode::inactiveProperty).toArray(BooleanProperty[]::new)));
+	/**
+	 * Bind this margin property to the sum of the children margin properties.
+	 */
+	public void bindMargins() {
```

Comment:
Yes, since this method is called to recalculate the bindings when undoing the collapsed node. See the `CollapsedNode`'s `onMouseClicked` method.